README.md

```diff
@@ -98,16 +98,16 @@ Important thing to note is that the service area only make sense if we have the
 
 In order to launch AWS Batch jobs, do the following:
 * Login into AWS Console and navigate to AWS Batch: https://console.aws.amazon.com/batch/
-* Go to "Job Definitions" section, click on "tile-generation-dev" item and then on the radio-button corresponding to the latest revision. 
+* Go to "Job Definitions" section, click on the job name and then on the radio-button corresponding to the latest revision. 
+  * Use `tile-generation-dev` for dev environment
+  * Use `tile-generation-prod` for production environment
 * Click Actions -> Submit job
 * On the next screen do the following:
   * Specify job name, which describes current job the best (no special requirements for the name)
-  * Specify `tile-generation-dev-queue` as the job queue name. 
-  * **Important**: specify **Array** as job type. 
-  * Specify number of jobs to be run as parallel in Array Size field. If you don't know how many you need, put `8`. It took 5-6 hours for 8 jobs to generate the tiles for entire service area up to maximum zoom level (18) during previous tests. 
-  * Navigate to **Environment variables** section and make sure that we have both mandatory variables set to correct values:
-    * `MAPNIK_TILE_S3_BUCKET`: should correspond with the target S3 bucket; `mbta-map-tiles-dev` for dev environment or `mbta-map-tiles` for production environment. If you want to upload files to different bucket, you can specify it here as well, but make sure that batch jobs have permissions to write there. 
-    * **Important**: `BATCH_JOB_COUNT`: should be equal to what you specified in Array Size field. Failure to set the right value here will result in either missing tiles or in overlap between the jobs (they are going to generate the same tiles).
-  * Click "Submit Job" and wait for jobs to run. You can also check Cloud Watch logs, while the jobs are running. Links to those logs are available from the Jobs page. 
+  * Specify `tile-generation-dev-queue` as the job queue name
```

Comment:
Should be something like "specify `tile-generation-dev-queue` for dev environment or `tile-generation-prod-queue` for production environment"

Reply:
Fixed